### DISCUSSION TITLE
Create another method to persist data generated by actions

### DISCUSSION THREAD

**Discussion by gabriel-logan** [reduxjs/redux-toolkit#4520] [Ideas]
Just like redux-persist, I wanted a lib to be created for this purpose of data persistence, but with the approval of the official creators of redux, redux-persist no one responds anymore and it seems like it was abandoned a long time ago.

I am available to help voluntarily in creating such a method, both for use in redux-toolkit and for use in redux legacy.

**Comment by phryneas** [comment]
You might also want to check out https://github.com/zewish/redux-remember - we heard good things about it as a replacement for persist, but don't have any experience of our own with it.

Maybe that already does what you need?
